pom.xml

```diff
@@ -222,13 +223,14 @@
       <version.hibernate.javax.persistence>1.0.0.Final</version.hibernate.javax.persistence>
       <version.hibernate.search>5.9.0.Final</version.hibernate.search>
       <version.hikaricp>2.4.6</version.hikaricp>
+      <version.javax.activation>1.1.1</version.javax.activation>
       <version.javax.cache>1.0.0</version.javax.cache>
       <version.jboss.logging>3.3.1.Final</version.jboss.logging>
-      <version.jboss.marshalling>2.0.2.Final</version.jboss.marshalling>
+      <version.jboss.marshalling>2.0.5.Final</version.jboss.marshalling>
       <version.jgroups>4.0.11.Final</version.jgroups>
       <version.jta>1.0.1.Final</version.jta>
       <version.lucene>5.5.5</version.lucene>
-      <version.maven-compiler-plugin>3.6.1</version.maven-compiler-plugin>
+      <version.maven-compiler-plugin>3.7.0-jboss-1</version.maven-compiler-plugin>
```

Comment:
What's changed in the jboss-1 version?

Reply:
It includes a patch for https://issues.apache.org/jira/browse/MCOMPILER-320

Reply:
Please add a comment in the POM